examples/platform/silabs/SiWx917/BUILD.gn

```diff
@@ -51,6 +51,7 @@ declare_args() {
 # Sanity check
 assert(chip_enable_wifi)
 silabs_plat_si91x_wifi_dir = "${chip_root}/src/platform/silabs/SiWx917/wifi"
+silabs_plat_dir = "${chip_root}/src/platform/silabs"
```

Comment:
move **silabs_plat_dir** above **silabs_plat_si91x_wifi_dir** and we can reuse this define?